library-best-practices/JLBP-2.md

```diff
@@ -0,0 +1,20 @@
+[JLBP-2] Minimize API surface
+-----------------------------
+
+- Try not to expose types from your dependencies.
+  - Your dependencies have their own rules about how often
+    they do major version bumps and make breaking changes,
+    and could be shorter than yours, which could cause a lot
```

Comment:
", and" --> . These might be shorter than yours which causes problems when you

Reply:
I split the sentences as suggested, but used slightly different wording.